mne/io/tests/test_raw.py

```diff
@@ -166,10 +166,13 @@ def _test_raw_reader(
             fnirs, atol = "hbo", 1e-10
         elif "hbr" in raw:
             fnirs, atol = "hbr", 1e-10
-        else:
-            assert "fnirs_cw_amplitude" in raw, "New channel type necessary?"
+        elif "fnirs_cw_amplitude" in raw:
             fnirs, atol = "fnirs_cw_amplitude", 1e-10
-        picks = pick_types(other_raw.info, meg=meg, eeg=eeg, fnirs=fnirs)
+        else:
+            assert "seeg" in raw, "New channel type necessary?"
```

Comment:
Let's help the next person who hits the same problem :)
```suggestion
            # e.g., https://github.com/mne-tools/mne-python/pull/11432/files
            assert "seeg" in raw, "New channel type necessary? See gh-11432 for example"
```